Doc/tutorial/introduction.rst

```diff
@@ -182,6 +182,29 @@ the first quote::
    >>> print(r'C:\some\name')  # note the r before the quote
    C:\some\name
 
+There is one subtle aspect to raw strings that is of special concern to Windows
+programmers:  a raw string may not end in an odd number of ``\`` characters.
+This is because the interpreter sees this as an escaped quote character, and so
+it does not recognize it as the end of the string::
+
+   >>> fn = r'C:\this\will\not\work\'
+     File "<stdin>", line 1
+       fn = r'C:\this\will\not\work\'
+                                    ^
+   SyntaxError: EOL while scanning string literal
+
+There are several workarounds for this.  One is to use regular strings and
+double the backslashes::
+
+   >>> fn = 'C:\\this\\will\\work\\'
+
+Another is to add a blank character before the quote and then remove it::
+
+   >>> fn = r'C:\this\will\work\ '.strip()
+
+Strings can be concatenated (glued together) with the ``+`` operator, and
+ repeated with ``*``::
```

Comment:
There's a `::` at the end here as if this is about to start a code block, but unless I'm missing something, I don't see any examples of using those being added.

Reply:
Line 205 and 206 aren't edited by me. They were already part of the documentation. They are showing in the PR because I removed a space behind them which I found unnecessary. I haven't made any changes to these lines other than this.